.github/workflows/code-workflows.yml

```diff
@@ -0,0 +1,65 @@
+name: Code workflows
+on: [push, pull_request]
+jobs:
+  lint-job:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Running py38-lint with tox
+        uses: ymyzk/run-tox-gh-actions@main
+        with:
+          tox-args:
+            -e py38-lint
+
+  test-job:
+    needs: lint-job
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Running tests for ${{ matrix.python-version }} with tox
+        uses: ymyzk/run-tox-gh-actions@main
+
+  # build-job:
+  #   needs: test-job
+  #   runs-on: ubuntu-20.04
+  #   strategy:
+  #     matrix:
+  #       python-version: [3.8]
+  #   steps:
+  #     - name: Check out repository code
+  #       uses: actions/checkout@v2
+  #     - name: Set up Python ${{ matrix.python-version }}
+  #       uses: actions/setup-python@v2
+  #       with:
+  #         python-version: ${{ matrix.python-version }}
+  #     - name: Running build for ${{ matrix.python-version }} with tox
+  #       uses: ymyzk/run-tox-gh-actions@main
+  #       with:
+  #         tox-args:
+  #           -e package
+  #     - name: Saving Artifacts.
+  #       uses: actions/upload-artifact@v2
+  #       with:
+  #         name: dql-executable
+  #         path: dist/dql
+  #     - name: Publish package
+  #       uses: pypa/gh-action-pypi-publish@27b31702a0e7fc50959f5ad993c78deac1bdfc29
+  #       with:
+  #         user: __token__
+  #         password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
Is there a reason to leave this comment around?

Reply:
I wanted to get some input from you about this. How do you release a version today? I would like to see if we can get that into a job that can be run with credentials.

Reply:
Okay, that'll be difficult for you to test since you don't have the credentials. I guess leave this in and I'll fiddle with it after it's merged to see if I can get it working